[rocBLAS] [hipBLAS] CHANGELOG update for 7.2 (#2668)
ROCm/rocm-libraries: projects/rocblas/CHANGELOG.md

rocBLAS documentation is available at
[https://rocm.docs.amd.com/projects/rocBLAS/en/latest/index.html](https://rocm.docs.amd.com/projects/rocBLAS/en/latest/index.html).

## rocBLAS 5.2.0 for ROCm 7.2

### Added

> * Level 3 syrk_ex function for both C and FORTRAN but without ILP64 form of API.

Reviewer: ```suggestion
* Level 3 `syrk_ex` function for both C and FORTRAN but without API support for the ILP64 format.
```

> ### Changed

Reviewer: ```suggestion
### Resolved issues
```

Reviewer: Can you move this section beneath Optimized?

> * Client memory use counts for environment ROCBLAS_CLIENT_RAM_GB_LIMIT were corrected and avoids false clang static analysis warnings.

Reviewer: ```suggestion
* Corrected client memory use counts for the `ROCBLAS_CLIENT_RAM_GB_LIMIT` environment variable.
* Fix to avoid false Clang static analysis warnings.
```

### Optimized

> * Level 2 tpmv function.

Reviewer: ```suggestion
* Level 2 `tpmv` function.
```
